### ISSUE TITLE
Knightlab timeline throws TypeError

### ISSUE THREAD

**Issue by ajstanley** [Daniel-KM/Omeka-S-module-Timeline#1]
A sample timeline comprised of images with dc:title, dc:description and dc:date works perfectly with Simile, but when the Knightslab library is called the javascript fails with
```
TypeError: this._el.container is null
timeline.js:13087:4
perf_trace {"name": "e2e", "parent": "PageEvents.BIGPIPE_ONLOAD"}
```
Module configuration is all default.

**Comment by Daniel-KM** [comment]
In fact, I didn't use the Knightlab's library, because I don't have an account on it. So the code is simply copied from the Omeka Classic version. So it will be hard to fix it for me.

**Comment by ajstanley** [comment]
I found the error - lines 70 and 73 of timeline_knightslab.phtml both add 'timeline-' to the div identifier.  This should only be done once.  I can put in a pull request, or you could edit it inline (easy fix).

Its an amazing module and works really well.  Thanks for this.

**Comment by Daniel-KM** [comment]
Thanks! (merged #2 )
